src/components/MenuBar/MenuBar.tsx

```diff
@@ -171,6 +175,17 @@ export default function MenuBar() {
     };
   }
 
+  if (isHostIn !== isHostInState) {
+    if (isHostIn) {
+      setNotification({ message: NOTIFICATION_MESSAGE.REPORTER_HAS_JOINED });
+      setIsHostInState(isHostIn);
+    } else if (!isHostIn) {
+      audioTrack?.disable();
```

Comment:
Where is the audioTrack enable? - After asking the participant if he wants to enable his audio.